.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,7 @@ Describe the issue that you're seeing.
 
 ### Steps to reproduce
 
-Clear steps describing how to reproduce the issue.
+Clear steps describing how to reproduce the issue. Please please please link to a demo project if possible, this makes your issue _much_ easier to diagnose (seriously).
```

Comment:
I love this. 😄

Reply:
😄 hopefully gets the point across that a reproducible demo is the number one thing someone can do to help getting their issue triaged.